core/dbt/include/global_project/macros/adapters/apply_grants.sql

```diff
@@ -0,0 +1,52 @@
+{% macro get_show_grant_sql(relation) %}
+{{ return(adapter.dispatch("get_show_grant_sql", "dbt")(relation)) }}
+{% endmacro %}
+
+{% macro default__get_show_grant_sql(relation) %}
+show grants on {{ relation.type }} {{ relation }}
+{% endmacro %}
+
+{% macro get_grant_sql(relation, grant_config) %}
+{{ return(adapter.dispatch('get_grant_sql', 'dbt')(relation, grant_config)) }}
+{% endmacro %}
+
+{% macro default__get_grant_sql(relation, grant_config) %}
+    {% for privilege in grant_config.keys() %}
+     {{ log('privilege: ' ~ privilege) }}
+        {% set grantees = grant_config[privilege] %}
+        {% for grantee in grantees %}
+            grant {{ privilege }} on {{ relation.type }} {{ relation }} to {{ grantee }};
+        {% endfor %}
+    {% endfor %}
+{% endmacro %}
+
+{% macro get_revoke_sql(relation, grant_config) %}
+{{ return(adapter.dispatch("get_revoke_sql", "dbt")(relation, grant_config)) }}
+{% endmacro %}
+
+{% macro default__get_revoke_sql(relation, grant_config) %}
+    {% for privilege in grant_config.keys() %}
+        {% set grantees = grant_config[privilege] %}
+        {% for grantee in grantees if grantee !=  target.user %}
+            revoke {{ privilege }} on {{ relation.type }} {{ relation }} from {{ grantee }}
+        {% endfor %}
+    {% endfor %}
+{% endmacro %}
+
+{% macro apply_grants(relation, grant_config, should_revoke) %}
+{{ return(adapter.dispatch("apply_grants", "dbt")(relation, grant_config, should_revoke=True)) }}
```

Comment:
We need to pass the actual argument value through, rather than always setting this to `True`:
```suggestion
{{ return(adapter.dispatch("apply_grants", "dbt")(relation, grant_config, should_revoke)) }}
```
(This doesn't set the default, it overrides the value being passed in!)

Reply:
oh okay thats interesting when we were first testing out it seemed like it was replacing the True we set via default macro call to a None value so good to know.